src/main/java/com/synopsys/integration/alert/workflow/startup/StartupManager.java

```diff
@@ -138,21 +136,13 @@ public void logConfiguration() {
 
         logger.info("----------------------------------------");
         logger.info("Alert Configuration: ");
-        logger.info("Logging level:           {}", alertProperties.getLoggingLevel().orElse(""));
+        logger.info("Alert Server URL:          {}", alertProperties.getServerUrl().orElse(""));
+        logger.info("Logging level:             {}", alertProperties.getLoggingLevel().orElse(""));
         logger.info("Alert Proxy Host:          {}", proxyHost.orElse(""));
         logger.info("Alert Proxy Port:          {}", proxyPort.orElse(""));
         logger.info("Alert Proxy Authenticated: {}", authenticatedProxy);
         logger.info("Alert Proxy User:          {}", proxyUsername.orElse(""));
         logger.info("");
-        logger.info("BlackDuck URL:                 {}", blackDuckProperties.getBlackDuckUrl().orElse(""));
```

Comment:
We aren't logging any of the Black Duck information anymore?

Reply:
BlackDuck isn't required any more.  We could perhaps iterate through the providers and log their fields.  Or loop through descriptors and log their configuration other than sensitive data.